web/src/components/TestSpec/Header.tsx

```diff
@@ -7,19 +7,24 @@ interface IProps {
   affectedSpans: number;
   assertionsFailed: number;
   assertionsPassed: number;
+  selector: string;
   title: string;
 }
 
-const Header = ({affectedSpans, assertionsFailed, assertionsPassed, title}: IProps) => {
+const Header = ({affectedSpans, assertionsFailed, assertionsPassed, selector, title}: IProps) => {
   return (
     <S.Column>
-      <Editor
-        type={SupportedEditors.Selector}
-        editable={false}
-        basicSetup={{lineNumbers: false}}
-        placeholder="Selecting All Spans"
-        value={title || 'All Spans'}
-      />
+      {title ? (
```

Comment:
I'm wondering if we want to show the selector on hover or something like that, should it be only displayed after clicking the test spec now?